zuul_alpha/core.py

```diff
@@ -149,10 +181,10 @@ def _encrypt(self, secret_name, secret):
         self._validate_secret(secret_name, secret)
         secret = secret.encode('utf-8')
 
-        pkcs1_pad = 42
+        OAEP_pad = 42
```

Comment:
It's the padding size of the encryptor standard. If it's changed to anything lower, then encrypts may fail for secrets near the payload size of 214 bytes for 2048-bit keys. 

I suppose we could protect from this by catching that Exception and using AES.